src/test/java/sirius/db/jdbc/SQLStringListPropertyEntitySpec.groovy

```diff
@@ -0,0 +1,55 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.jdbc
+
+import sirius.kernel.BaseSpecification
+import sirius.kernel.di.std.Part
+
+class SQLStringListPropertyEntitySpec extends BaseSpecification {
+
+    @Part
+    static OMA oma
+
+    def "test many list entries"() {
```

Comment:
add a test with a short string field and many list items so that data truncation happens and ensure that a proper and understandable exception is shown / logged